frontend/app/components/cc-data-table/template.hbs

```diff
@@ -1,4 +1,4 @@
-<table class="uk-table uk-table-divider uk-table-responsive">
+<table class="cc-data-table uk-table uk-table-divider uk-table-responsive uk-table-hover uk-table-striped ">
```

Comment:
please use uikit class `uk-table-hover` (like you already did) and set the uikit default var `@table-hover-row-background` to you color in `app.scss`. Note to set your var before importing uikit.

I would like to avoid custom scss as much as possible.